src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -680,6 +682,51 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
             }]
         });
 
+        let coverage = env::var_os("CI_COVERAGE_ENABLED").is_some();
+
+        let env = if coverage {
+            Some(vec![EnvVar {
+                name: "LLVM_PROFILE_FILE".to_string(),
+                value: Some("/coverage/clusterd-%p-%9m%c.profraw".to_string()),
```

Comment:
`clusterd` here looks inappropriately hardcoded. Perhaps blat in the service name instead?